tedium.js

```diff
@@ -203,13 +290,14 @@ Promise.resolve().then(() => {
   const promises = [];
 
   for (const ghRepo of ghRepos) {
-    promises.push(rateLimit(100).then(() => {
+    promises.push(Promise.resolve().then(() => {
```

Comment:
Looks like passing a function into Promise.resolve resolves the promise with the function as the promise's value:

```javascript
Promise.resolve(function() {return 1}).then(function(r) { console.log(r)}); // logs the function, not 1
```

In general I like to put as much code related to a promise inside the then() of a Promise.resolve in order to ensure that errors are properly tracked with that promise.